python/foxglove-sdk-examples/live-visualization/main.py

```diff
@@ -51,7 +50,8 @@
 
 class ExampleListener(ServerListener):
     def __init__(self) -> None:
-        self.subscribers: set[int] = set()
+        # Map client id -> set of subscribed topics
+        self.subscribers: dict[int, set[str]] = {}
```

Comment:
```suggestion
        self.subscribers: dict[int, set[str]] = defaultdict(set)
```

Add `from collections import defaultdict`

And then you can just do `self.subscribers[client.id].add(topic)`, the set will be created if one doesn't exist yet.

Reply:
👍 but I went with the `setdefault` suggestion below